### PR TITLE
Document the fact that entity fragments exists

### PR DESCRIPTION
**Pull request by Irma12**
https://deploy-preview-1115--roadie.netlify.app/docs/details/decorating-components/

### CODE REVIEW

**Comment by dtuite** [comment]
I sent Irma a loom with some thoughts

File: content/docs/details/decorating-components/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Introduction
+publishedDate: '2023-09-05'
+description: An introduction about decorating components.
+---
+
+Catalog is built around the concept of [entity descriptor YAML files](https://backstage.io/docs/features/software-catalog/descriptor-format/), therefore, any change you wish to make to your components, needs to be done via editing those files.
```
Comment (Xantier) [inline content/docs/details/decorating-components/index.md:7]: This isn't quite true. The catalog doesn't depend on YAML files but can be populated using any methods

File: content/docs/details/decorating-components/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: Introduction
+publishedDate: '2023-09-05'
+description: An introduction about decorating components.
+---
+
+Catalog is built around the concept of [entity descriptor YAML files](https://backstage.io/docs/features/software-catalog/descriptor-format/), therefore, any change you wish to make to your components, needs to be done via editing those files. 
+
+However, as YAML files can grow in complexity, they can become challenging to read and maintain. Editing those files, can sometimes be a long process, especially if ownership is shared accross the teams in your organization.
```
Comment (Xantier) [inline content/docs/details/decorating-components/index.md:9]: It's likely better to take an approach where we talk about multiple sources of data.

File: content/docs/details/decorating-components/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+![Add link](./add_link.png)
+
+New link properties need to follow [link properties types](https://backstage.io/docs/features/software-catalog/descriptor-format#links-optional). Usually semantics of the type field are undefined, but we have added few predefined options you can select from a dropdown list.
+
+Adding annotations is done via 'Add annotation' button in 'Annotations' section.
+
+![Add annotation](./add_annotation.png)
+
+New annotation properties need to follow [annotation properties types](https://backstage.io/docs/features/software-catalog/descriptor-format#annotations-optional). 
+You will find all existing annotations used in components accross catalog in a dropdown list.
```
Comment (Xantier) [inline content/docs/details/decorating-components/index.md:38]:
```suggestion
You will find all existing annotations used in components across the catalog in a dropdown list.
```

File: content/docs/details/decorating-components/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+When you are happy with added annotations or/and links simply click 'Save' button and you will shortly see you changes.
```
Comment (Xantier) [inline content/docs/details/decorating-components/index.md:40]:
```suggestion
When you are happy with added annotations and/or links simply click 'Save' button and you will shortly see you changes.
```

File: content/docs/details/decorating-components/index.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+title: Introduction
+publishedDate: '2023-09-05'
+description: An introduction about decorating components.
+---
+
+Components within the software catalog are structured around [metadata YAML files](https://backstage.io/docs/features/software-catalog/descriptor-format/) alongside the code, which are then collected and presented in Backstage for visualization. The source of truth for the components are mentioned YAML files, stored in source control such as GitHub, GitHub Enterprise, GitLab, Bitbucket, and others.
```
Comment (dtuite) [inline content/docs/details/decorating-components/index.md:7]:
```suggestion
Decorators are an easy way to add links and annotations to Entities, without editing the YAML file that the Entity originates from.

Decorators are stored inside Roadie and available via the API.
```

File: content/docs/details/decorating-components/index.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+title: Introduction
+publishedDate: '2023-09-05'
+description: An introduction about decorating components.
+---
+
+Components within the software catalog are structured around [metadata YAML files](https://backstage.io/docs/features/software-catalog/descriptor-format/) alongside the code, which are then collected and presented in Backstage for visualization. The source of truth for the components are mentioned YAML files, stored in source control such as GitHub, GitHub Enterprise, GitLab, Bitbucket, and others. 
+
+Because the process of adding components to the catalog can be achieved through manual registration, creating new components within Backstage, or integrating with an external source, the catalog may often comprise components with different sources of data. Consequently, modifying these components can be challenging, as it involves making changes in multiple locations.
```
Comment (dtuite) [inline content/docs/details/decorating-components/index.md:9]:
```suggestion
```

We don't need to explain why the feature exists in technical documentation.

File: content/docs/details/decorating-components/index.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+Idea behind component decorator is to provide an easy way to modify components in one place, through Backstage, and enhance usability for components used accross organization.
```
Comment (dtuite) [inline content/docs/details/decorating-components/index.md:11]:
```suggestion
```

**Comment by Irma12** [comment]
> Let's simplify the introduction and get people to the good stuff quicker.

I removed all additional stuff, as you requested

**Review by dtuite — APPROVED** [review]

**Pull request merged**